Makefile

```diff
@@ -125,12 +125,15 @@ test-silent:
 	$(eval undefine VERBOSE)
 	${GOTEST} -tags unit -timeout=30s ./agent/...
 
-run-integ-tests: test-registry gremlin container-health-check-image run-sudo-tests
+run-integ-tests: test-registry gremlin container-health-check-image run-sudo-tests run-sudo-integ-tests
 	ECS_LOGLEVEL=debug ${GOTEST} -tags integration -timeout=25m ./agent/...
 
 run-sudo-tests:
 	sudo -E ${GOTEST} -tags sudo -timeout=10m ./agent/...
 
+run-sudo-integ-tests:
+	sudo -E ${GOTEST} -tags 'sudo integration' -timeout=10m ./agent/...
```

Comment:
i think the new test takes more than 1 min to finish, so you should probably bump timeout here.. e.g. 12m

Reply:
Okay.

Reply:
nvm seems like the test was hanging on task cleanup, otherwise it should just take a few seconds. please fix cleanup instead of bumping timeout